chapters/VK_EXT_debug_utils.txt

```diff
@@ -42,7 +42,11 @@ fname:vkSetDebugUtilsObjectNameEXT as defined below.
 --
 include::{generated}/api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
-  * pname:device is the device that created the object.
+  * pname:device is the device that created the object, or, if the device is a
```

Comment:
```suggestion
  * pname:device is the device that created the object, or, if the object is a
```
Was this intended to be `object` rather than `device`? I'm having difficultly parsing the sentence otherwise

Reply:
Oops, indeed.